ui/sdl2.c

```diff
@@ -149,12 +182,12 @@ static void sdl_update_caption(struct sdl2_console *scon)
     }
 
     if (qemu_name) {
-        snprintf(win_title, sizeof(win_title), "QEMU (%s-%d)%s", qemu_name,
+        snprintf(win_title, sizeof(win_title), "XQEMU (%s) (%s-%d)%s", QEMU_PKGVERSION, qemu_name,
                  scon->idx, status);
-        snprintf(icon_title, sizeof(icon_title), "QEMU (%s)", qemu_name);
+        snprintf(icon_title, sizeof(icon_title), "XQEMU (%s) (%s)", QEMU_PKGVERSION, qemu_name);
     } else {
-        snprintf(win_title, sizeof(win_title), "QEMU%s", status);
-        snprintf(icon_title, sizeof(icon_title), "QEMU");
+        snprintf(win_title, sizeof(win_title), "XQEMU (%s)%s", QEMU_PKGVERSION, status);
+        snprintf(icon_title, sizeof(icon_title), "XQEMU (%s)", QEMU_PKGVERSION);
```

Comment:
nit: Yikes, so many variations..

I suggest fixing this "QEMU" mess by adding a seperate variable name `project_title` or something.

Just out of interest: Do you know what these variations do?

Reply:
Two pairs for title bar and window switcher, with and without `name` as provided by the `-name` flag during startup.